force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -80,16 +99,20 @@ public with sharing class ClassesObjectsMethods {
      * @return A string consisting of only unique words from the input sentence.
      */
     public static String removeDuplicateWords(String sentence) {
+       
         // Split the sentence into words
+        List<String> words = sentence.split(' ');
 
         // Create a set to store the unique words
-
         // Add each word to the set
+        Set<String> wordSet = new Set<String>();
```

Comment:
Here, you might want to use a more descriptive variable name: `uniqueWords` or `uniqueWordSet`.